tipg/db.py

```diff
@@ -1,31 +1,36 @@
 """tipg.db: database events."""
 
-from pathlib import Path
+import os
+import pathlib
 from typing import Any, Optional
 
 import orjson
 from buildpg import asyncpg
 
 from tipg.dbmodel import get_collection_index
+from tipg.errors import FunctionDirectoryDoesNotExist
 from tipg.settings import PostgresSettings
 
 from fastapi import FastAPI
 
+custom_sql: list[pathlib.Path] = []
+user_sql_dir = os.environ.get("TIPG_CUSTOM_SQL_DIRECTORY", None)
+if user_sql_dir:
+    if not pathlib.Path(user_sql_dir).exists():
+        raise FunctionDirectoryDoesNotExist
+
+    custom_sql = list(pathlib.Path(user_sql_dir).glob("*.sql"))
```

Comment:
This is just simpler, we didn't really need this to be in the `PostgresSettings` because it's more an TiPG settings.

Reply:
The approach is fine, but I think we should still do this somewhere in settings.py so that all the settings can be found in one place for anyone who is trying to look for all the settings they can use.